scripts/lint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+function mark_unstable {
+    java -jar jenkins-cli.jar -s $JENKINS_URI set-build-result unstable
+}
+
+set -x
+trap 'mark_unstable' ERR
```

Comment:
I sort of don't like that this puts some Jenkins-specific stuff directly into this script, but I tried doing something like `./scripts/lint.sh || java -jar jenkins-cli.jar -s $JENKINS_URI set-build-result unstable` and it didn't want to work.